Dockerfile

```
@@ -0,0 +1,12 @@
+FROM alpine:3.10
+LABEL maintainer "Wes Lambert, wlambertts@gmail.com"
+LABEL version="CyberChef server 0.0.1"
+LABEL description="Dockerized version of Cyberchef server (from the fine folks at GCHQ: https://github.com/gchq/CyberChef-server)"
```

Please could you change this to
```suggestion
LABEL description="Dockerised version of Cyberchef server ( https://github.com/gchq/CyberChef-server)"
```

---

Certainly!

---

Updated in PR.